providers/gce/gce_fake.go

```diff
@@ -48,7 +49,8 @@ func DefaultTestClusterValues() TestClusterValues {
 		ZoneName:          "us-central1-b",
 		SecondaryZoneName: "us-central1-c",
 		ClusterID:         "test-cluster-id",
-		ClusterName:       "Test Cluster Name",
+		ClusterName:       "Test-Cluster-Name",
```

Comment:
why do you change the clkustername?

Reply:
It appears it selflinks, so I don't think spaces are allowed. This confused me, and I used ClusterID instead of ClusterName at first.

Reply:
til